doc/source/development/contributing.rst

```diff
@@ -299,6 +299,11 @@ We'll now kick off a three-step process:
    python setup.py build_ext -j 4
    python -m pip install -e . --no-build-isolation --no-use-pep517
 
+.. note::
+    If the ``python setup.py build_ext -j 4`` command fails on a linux distribution you might
```

Comment:
can we be more specific than "you might be able to"?  is there reason to believe this is a common failure mode?